platform/ios/test/MGLLayerTests.m

```diff
@@ -0,0 +1,25 @@
+#import "MGLMapViewTests.h"
```

Comment:
Moved the files and added to macos as well. Excluded the custom layer tests for macos as the header is ios specific. Also changed the source test according to https://github.com/mapbox/mapbox-gl-native/pull/7203#discussion_r90384300